### PR TITLE
Update bugzilla and errata scrapers to query on last_modified

### PR DESCRIPTION
**Pull request by yashvardhannanavati**
Update bugzilla and errata scrapers to query on last_modified rather than creation_date

### CODE REVIEW

File: scrapers/bugzilla.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def get_bugzilla_bugs(self, start_date):
             LEFT JOIN bugzilla.profiles AS assigned ON bugs.assigned_to = assigned.userid
             LEFT JOIN bugzilla.profiles AS reported ON bugs.reporter = reported.userid
             LEFT JOIN bugzilla.profiles AS qa ON bugs.qa_contact = qa.userid
-            WHERE classifications.name = 'Red Hat' AND bugs.creation_ts > '{}'
+            WHERE classifications.name = 'Red Hat' AND bugs.lastdiffed > '{}'
```
Comment (mprahl) [inline scrapers/bugzilla.py]: I think we actually want to use `delta_ts` here.

Comment (yashvardhannanavati) [reply scrapers/bugzilla.py]: They are the same for all bugs :)

Comment (mprahl) [reply scrapers/bugzilla.py]: Ah I see. This is the value that's exposed in the API, so I think we should use that one instead.

Comment (yashvardhannanavati) [reply scrapers/bugzilla.py]: Agreed. Changed it :)

**Comment by mprahl** [comment]
:+1:

**Pull request merged**